{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Base Image
+FROM python:3.6.6-alpine3.7
+
+# Best practice: if you build contains several layers you should order them
+# from the less frequently changed to the more frequently changed (caching)
+
+# Install tools required for the project
+# Best practice:
+# 1. do not install unnecessary packages, keep it small.
+# 2. keep an alphabetic order to avoid duplicates
+RUN apk add --no-cache --update \
+    build-base \
+    git \
+    libffi-dev \
+    linux-headers \
+    pcre-dev \
+    postgresql-client \
+    python3-dev
+
+# Run as non root user (best practice)
+ENV USERNAME=syapse
```

Comment:
Is there any value in being flexible here?

Reply:
no really. assuming that once I picked one username then I will *never* change it. avoiding to change  _n_ lines manually.